CHANGELOG.md

```diff
@@ -14,6 +14,11 @@
 #### New Features
 
 #### _**Note**_: Breaking changes in the API
+* Refactoring #3654: 
+    * Removed deprecated KubernetesClient.customResource / RawCustomResourceOperationsImpl, please use the generic resource api instead 
+    * Removed deprecatedHttpClientUtils.createHttpClient(final Config config, final Consumer<OkHttpClient.Builder> additionalConfig), please use the OkHttpClientFactory instead
+    * Removed deprecated methods on SharedInformerFactory dealing with the OperationContext
+* Refactoring #2838: Readiness/OpenShiftReadiness moved from client.internal.readiness to client.readiness
```

Comment:
Could you please update migration v6 guide as well?

https://github.com/fabric8io/kubernetes-client/blob/master/doc/MIGRATION-v6.md

Reply:
Sure, just wanted to see if everyone was okay with this breaking change, or if we'd want to leave behind deprecated versions in the internal packages.